int/docker-compose.yml

```diff
@@ -51,7 +71,7 @@ services:
       dockerfile: ./Dockerfile.test.runner
     environment:
       - WAIT_BEFORE_HOSTS=3
-      - WAIT_HOSTS=acapy_pickup:3000,echo:3002
+      - WAIT_HOSTS=acapy_pickup:3000,echo:3002,redis:6379
```

Comment:
is adding redis to the wait_hosts needed with the health check?

Reply:
No it is not, I think this was added in to check that the server was connecting properly

Reply:
We need to stick with either health checks or wait scripts. We've got healthchecks so I think we should commit to that, making these `WAIT_` vars unneeded. Dockerfile for the juggernaut would also need to be updated to drop the addtion of the wait script.